spec/values/calculation/round/one_argument.hrx

```diff
@@ -0,0 +1,151 @@
+<===> README.md
+Most of the same behavior tested for `calc()` applies to `round()`, but for
+terseness' sake isn't tested explicitly.
+
+<===>
+================================================================================
+<===> zero/input.scss
+a {b: round(0)}
+
+<===> zero/output.css
+a {
+  b: 0;
+}
+
+<===>
+================================================================================
+<===> positive/input.scss
+a {b: round(1)}
+
+<===> positive/output.css
+a {
+  b: 1;
+}
+
+<===>
+================================================================================
+<===> negative/input.scss
+a {b: round(-5.6)}
+
+<===> negative/output.css
+a {
+  b: -6;
+}
+
+<===>
+================================================================================
+<===> operation/preserves_units/input.scss
+a {b: round(-7px / 4em) * 1em}
+
+<===> operation/preserves_units/output.css
+a {
+  b: -2px;
+}
+
+<===>
+================================================================================
+<===> preserved/variable/input.scss
```

Comment:
Since a single-argument needs simplifiable parameters, I separated this test. (Check line #116)